gravity.c

```diff
@@ -90,18 +94,21 @@ bool gravityDB_getTable(unsigned char list)
 	return true;
 }
 
-const char* gravityDB_getDomain(void)
+inline const char* gravityDB_getDomain(void)
```

Comment:
Is there a reason for inlining?

Reply:
This routine is basically a wrapper for `sqlite3_step()` and `sqlite3_column_text()` (alongside some error checking). It is called extremely often - once for each blocking domain, so it might be called even millions of times if users have huge blocking lists - so overhead should be avoided here. The compiler itself might decide to `inline` this function. I use this keyword here to support this decision and remove the function calling overhead.